packages/renderer/src/lib/ui/ProviderInfoCircle.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
 import { providerColors } from './ProviderInfoCircle';
 
-let { type }: { type: 'kubernetes' | 'podman' | 'docker' | undefined } = $props();
+let { type }: { type: 'kubernetes' | 'podman' | 'docker' | 'lima' | undefined } = $props();
```

Comment:
question (non-blocking): same as https://github.com/podman-desktop/podman-desktop/pull/10447/files#r1893707560

Reply:
In this case it is the serverType as input, and that string is thrown into providerType

Which is different from ProviderInfo, because there connectionType is used as input